IOSExercises/Views/DetailsViewController.m

```diff
@@ -0,0 +1,341 @@
+//
+//  DetailsViewController.m
+//  IOSExercises
+//
+//  Created by Arnold Joseph Caesar Esteban on 11/6/20.
+//  Copyright © 2020 Arnold Joseph Caesar Esteban. All rights reserved.
+//
+
+#import "DetailsViewController.h"
+
+@interface DetailsViewController() <UIPopoverPresentationControllerDelegate>
+
+@property (strong, nonatomic) IBOutlet UILabel *colorLabel;
+
+@property (strong, nonatomic) IBOutlet UILabel *birthdayLabel;
+
+@property (strong, nonatomic) IBOutlet UIButton *birthdayInfoButton;
+
+@property (strong, nonatomic) UIViewController *popoverViewController;
+
+@property (strong, nonatomic) UIDatePicker *datePicker;
+
+@end
+
+@implementation DetailsViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    [self initDefaultValue];
```

Comment:
Avoid reserved keywords like `init` (since `init` signifies that an instance is being created). In this case, `setup` seems to be the better prefix